digits/job.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, name, username, persistent = True):
         self._id = '%s-%s' % (time.strftime('%Y%m%d-%H%M%S'), os.urandom(2).encode('hex'))
         self._dir = os.path.join(config_value('jobs_dir'), self._id)
         self._name = name
+        self.group = group
```

Comment:
question: shall we increment `PICKLE_VERSION` and set `group='' ` in `__setstate__()` for earlier versions? I see that in `json_dict()` your are checking whether the attribute `group` exists. I am wondering what is best between setting all fields when loading picked jobs (and using default values when loading older jobs that don't have the new field) or checking later on at run time whether the field exists... I have a tiny inclination towards the former method but I'm curious to know what you think.